mirage/psf/segment_psfs.py

```diff
@@ -59,6 +59,10 @@ def generate_segment_psfs(ote, segment_tilts, out_dir, filters=['F212N', 'F480M'
     fov_pixels : int, optional
         Size of the PSF to generate, in pixels. Default is 1024.
 
+    boresight: list, optional
+        Telescope boresight offset in V2/V3. This offset is added on top of the individual 
```

Comment:
Are users going to know that the input boresight offset values should be in units of arcmin? Or should that info be added here to the documentation?